File: 2748-CalculateDelayedArrivalTime/2748-CalculateDelayedArrivalTime.cpp
// Last updated: 01/09/2025, 00:59:02
class Solution {
public:
    int findDelayedArrivalTime(int arrivalTime, int delayedTime) {
        int k=arrivalTime+delayedTime;
        if(k<24) return k;
        else if(k==24) return 0;
        else return abs(24-k);
    }
};

File: 2748-CalculateDelayedArrivalTime/2748-CalculateDelayedArrivalTime.py
# Last updated: 01/09/2025, 00:58:59
class Solution:
    def findDelayedArrivalTime(self, arrivalTime: int, delayedTime: int) -> int:
        k=arrivalTime+delayedTime
        if(k<24):
            p=k
            return k
        elif(k==0): 
            return 0
        else:
            p=abs(24-k)
            return p